knowledge-portal/source/README.md

```diff
@@ -11,7 +11,7 @@ You will have to modify the existing `gitlab-ci.yml` file to include:
         - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/source/.gitlab-ci.yml'
 
 ```
-## Configure CI/CD variables
+## Configure CI/CD variables (Deprecated)
```

Comment:
CI/CD variable configuration would still be required for the overall gitlab instance. So can we mention that in brackets, rather than Deprecated?

Reply:
done